### PR TITLE
Home task 5+Home task 6

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: ochubryk/Home-task_5/list.html [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+			width: 400px;
+			max-height: 4em;
+			overflow: auto;
+		}
+		</style>
+	</HEAD>
+	<BODY>
+		<div class="listmain">
+			<ul class="biglist">
+				<li>Контейнер с кодом</li>
```
Comment (pluhin) [inline ochubryk/Home-task_5/list.html]: Wrong nested list

File: ochubryk/Home-task_5/list.html [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+					<ul class="list">
+						<li>Цвет фона: черный</li>
+						<li>Цвет текста: зеленый</li>
+						<li>Ширина: 400рх</li>
+						<li>Максимальная высота контейнера: 4-е строки</li>
+						<li>Переполнение: автоматическое</li>
+					</ul>	
+				<li>Список</li>
+					<ul class="list">
```
Comment (pluhin) [inline ochubryk/Home-task_5/list.html:40]: wrong nested list position

**Review by pluhin — CHANGES_REQUESTED** [review]

File: ochubryk/Home-task_5/list.html (hunk 1 above)
Comment (ghost) [reply ochubryk/Home-task_5/list.html]: Fixed

File: ochubryk/Home-task_5/list.html (hunk 2 above)
Comment (ghost) [reply ochubryk/Home-task_5/list.html:40]: Fixed

**Review by pluhin — APPROVED** [review]
Good, approved

**Pull request merged**